user-docs/modules/ROOT/pages/fdo-glossary.adoc

```diff
@@ -10,7 +10,7 @@ Learn more about the FIDO device onboarding terminology.
 | FDO                    | FIDO Device Onboarding.
 | Device                 | Any hardware, device, or computer.
 | Device Credential (DC) | Key credential and rendezvous stored in the device at manufacture.
-| Keys                   | Keys to configure the manufacturing server
+| Keys                   a| Keys to configure the manufacturing server
```

Comment:
Is this a intended, same with the one below on the OV section.

Reply:
Yeah, that's intended. It makes the table cell to be processed so the list items are rendered correctly.